src/Monolog/Formatter/LogglyFormatter.php

```diff
@@ -37,7 +37,7 @@ public function __construct($batchMode = self::BATCH_MODE_NEWLINES, $appendNewli
      */
     public function format(array $record)
     {
-        if (isset($record["datetime"]) && ($record["datetime"] instanceof \DateTime)) {
+        if (isset($record["datetime"]) && ($record["datetime"] instanceof \DateTimeInterface)) {
```

Comment:
Oh my bad, I hadn't checked up on that

On 1:42PM, Fri, Feb 20, 2015 Christian Flothmann <notifications@github.com>
wrote:

> In src/Monolog/Formatter/LogglyFormatter.php
> <https://github.com/Seldaek/monolog/pull/511#discussion_r25067178>:
>
> > @@ -37,7 +37,7 @@ public function __construct($batchMode = self::BATCH_MODE_NEWLINES, $appendNewli
> >       */
> >      public function format(array $record)
> >      {
> > -        if (isset($record["datetime"]) && ($record["datetime"] instanceof \DateTime)) {
> > +        if (isset($record["datetime"]) && ($record["datetime"] instanceof \DateTimeInterface)) {
>
> This won't work with PHP 5.3 and 5.4. The DateTimeInterface was
> introduced in PHP 5.5.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Seldaek/monolog/pull/511/files#r25067178>.
>